asf_heat_pump_suitability/pipeline/run/heat_network_city_centres.py

```diff
@@ -55,13 +55,15 @@ def parse_arguments() -> argparse.Namespace:
         )
         uprn_gdf = uprns.generate_gdf_uprn_coords(uprn_df)
 
+        """ Existing heat network zones """
+
         # Load Plymouth existing heat network zone polygons
         print("Loading heat network zone data for Plymouth Local Authority...")
         plymouth_hn_zones_gdf = load_geodata.load_gdf_heat_network_zones(
             local_authority="plymouth"
         )
 
-        # Filter for UPRNs in existing heat network zones
+        # Label UPRNs in existing heat network zones
```

Comment:
I wonder if it might be useful to clarify
```suggestion
        # Label UPRNs in existing, potential and planned heat network zones
```